Source/UI/SecurityCodeComponent/SecurityCodeComponentConfiguration.swift

```diff
@@ -0,0 +1,49 @@
+// 
+//  SecurityCodeConfiguration.swift
+//  
+//
+//  Created by Okhan Okbay on 05/10/2023.
+//
+
+import Checkout
+import UIKit
+
+/**
+ Lets you configure and style the SecurityCodeComponent
+
+ - serviceAPIKey: The API Key you receive from checkout.com
+ - environment: Production or sandbox
+ - cardScheme: Optional card scheme
+    - If provided, card scheme's validation rules apply
+    - If not provided, security code is treated as valid for 3 and 4 digits
+ - style: Security Code Component wraps a text field in a secure way.
+ To style the inner properties like font, textColor etc, you must alter the style.
+ */
+
+public struct SecurityCodeComponentConfiguration {
+  let serviceAPIKey: String
+  let environment: Environment
+  let cardScheme: Card.Scheme?
```

Comment:
For Android it is working as below to inject CardScheme enum
`val cvvComponentConfig = CVVComponentConfig(
        cardScheme = CardScheme.fromString(cardSchemeValue = "VISA"),
        onCVVValueChange = { isValidCVV ->
            enteredVisaCVVUpdated.value = isValidCVV
        },
        cvvInputFieldStyle = inputFieldStyle,
    )`

Reply:
gotcha! will have a look into this to match Android

Reply:
I think we agreed on this offline and made the necessary changes. Anything still missing? @chintan-soni-cko

Reply:
No, good for me currently.